stitches/dev_matchup.py

```diff
@@ -0,0 +1,155 @@
+from stitches.pkgimports import *
+
+###############################################################################
+# Contains first draft of the functions used to match up, note that there are
+# some helper functions that will most likely be deprecated over time.
+# But I was struggling to figure out how to have functions work together
+# that are defined in different py scripts :(
+
+
+def cleanup_main_tgav(f):
+    """ This function cleans up the main_tgav_all_pangeo_list_models.csv to prepare it for match up, temp use while
+       the archive is still under devlopment.
+
+        :param f:         The path to the file to import, should be
+        ./stitches/data/created_data/main_tgav_all_pangeo_list_models.csv
+        :param save_individ_tgav:       True/False about whether to save the individual ensemble's
+            Tgav in its own file.
+            TODO: make save path an argument. Also note that this will be removed/changed when the archive structure
+            is finalized.
+
+        :return:                    A formatted data frame of Tgav time series for the netcdf file, including metadata.
+       """
+    if not (os.path.isfile(f)):
+        raise TypeError(f"file does not exist")
+
+    df = pd.read_csv(f)
+    # Select the columns containing actual data
+    df = df[['activity', 'model', 'experiment', 'ensemble_member', 'timestep', 'grid_type', 'file', 'year', 'tgav']]
+    # Rename columns and add a variable column
+    df = df.rename(columns={'activity':'activity', 'model':'model', 'experiment':'experiment',
+                            'ensemble_member':'ensemble', 'timestep':'timestep',
+                            'grid_type':'grid_type', 'file':'file', 'year':'year', 'tgav':'value'})
```

Comment:
Yes, the only thing that is being renamed is the ensemble_memeber and tgav file, I imagine this step would happen while the archive is being created so that the archive saved data with the ensemble, value, variable column names.